mmpose/models/heads/deeppose_regression_head.py

```diff
@@ -26,7 +26,8 @@ def __init__(self,
                  num_joints,
                  loss_keypoint=None,
                  train_cfg=None,
-                 test_cfg=None):
+                 test_cfg=None,
+                 out_sigma=False):
```

Comment:
I suggest placing `out_sigma` before `train_cfg`. It should not introduce any bc-breaking.

Reply:
Right, splitting `args`  and `cfg_args` is better

Reply:
I will do it when I finish the review :)

Reply:
Ok, thank you~